internal/controller/postgrescluster/suite_test.go

```diff
@@ -6,30 +6,19 @@ package postgrescluster
 
 import (
 	"context"
-	"os"
-	"strings"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"k8s.io/apimachinery/pkg/util/version"
-	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	"github.com/crunchydata/postgres-operator/internal/logging"
 	"github.com/crunchydata/postgres-operator/internal/testing/require"
 )
 
 var suite struct {
 	Client client.Client
-	Config *rest.Config
```

Comment:
Oh, the only use of ServerVersion was in the test in `internal/controller/postgrescluster/controller_test.go` where we were checking for older versions of k8s than we care about, ah